gradle/verification-metadata.xml

```diff
@@ -353,6 +354,42 @@
             <sha256 value="be46341a814e786c14c5d0e6cf5d470e55fff468b99509437c4014d8b2a20ce4" origin="Generated by Gradle because artifact wasn't signed"/>
          </artifact>
       </component>
+      <component group="com.diffplug.durian" name="durian-collect" version="1.2.0">
+         <artifact name="durian-collect-1.2.0.jar">
+            <pgp value="8756c4f765c9ac3cb6b85d62379ce192d401ab61"/>
```

Comment:
You added `^com[.]diffplug($|([.].*))` to the `8756c4f765c9ac3cb6b85d62379ce192d401ab61` trusted key, so adding the `com.diffplugin` entries here shouldn't be necessary. Does it fail without?

Reply:
I just ran the generation task, if it's not necessary, why does the generate task add them? Sounds like a bug?

Reply:
There are cases where the automated generation is not able to reduce properly.

Reply:
Fixed, and everything seems to work fine